src/main/java/com/exadel/ipromise/controllers/UserController.java

```diff
@@ -0,0 +1,79 @@
+package com.exadel.ipromise.controllers;
+
+import com.exadel.ipromise.dto.UserDto;
+import com.exadel.ipromise.service.UserService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.servlet.http.HttpSession;
+
+@RestController
+public class UserController {
```

Comment:
That's UserController and all your endpoints starts from "user", you can move this common part to the controller level

Reply:
Done